INSTALL.md

```diff
@@ -146,44 +118,50 @@ If you follow the below steps on greed field, configured, Red Hat CloudForms app
 
 *   A pre-configured catalog and associated service dialog are provided as part of the miq-RedHat-Satellite6 project on Github.  Complete the following steps to import these items.
 
-
-## Install cfme-rhconsulting-scripts per instructions:
-
-
+## Install cfme-rhconsulting-scripts per instructions on CFME appliance (this provides needed importing and exporting tools):
 
 *   [https://github.com/rhtconsulting/cfme-rhconsulting-scripts#install](https://github.com/rhtconsulting/cfme-rhconsulting-scripts#install)
 
+## Import Service Dialog and Catalog Items (these steps get done also on CFME appliance):
 
-## Import Service Dialog and Catalog Items:
-
-
-
-*   Clone miq-RedHat-Satellite6 repository
+*   change to then git clone miq-RedHat-Satellite6 repository to a working directory
     *   git clone https://github.com/RedHatOfficial/miq-RedHat-Satellite6.git
-*   Import Service Dialog
-    *   miqimport service_dialogs miq-RedHat-Satellite6/Dialogs/
-*   Import Catalog
-    *   miqimport service_catalogs miq-RedHat-Satellite6/Catalogs/
-
+*   Run the miqimport tool to import the provided Service Dialog(s) changing <VER> with v5.8 (CF 4.5) or v5.9 (CF 4.6)
+    *   miqimport service_dialogs miq-RedHat-Satellite6/Dialogs/<VER>
+*   Run the miqimport tool to import the provided Catalog item(s) changing <VER> with v5.8 (CF 4.5) or v5.9 (CF 4.6)
+    *   miqimport service_catalogs miq-RedHat-Satellite6/Catalogs/<VER>
 
 # Tagging:
 
-
 ## Tag Categories:
 
-
 ### Environment Tag Category
-
+The default environment tag category that ships with CFME does not allow multiple values so we need to delete and recreate it.  This is don in the Region configuration area of CFME.
+This tag will be populated with all the lifecycle environments from Satellite and used to tag what lifecycle environents can dbe used with each provider
 *   Delete environment Tag Category
 *   Recreate environment Tag Category
     *   Name: environment
     *   Description: Environment
     *   Long Description: Environment
     *   Show in Console: Yes
     *   Single Value: No
+    *   Capture C & U Data by Tag: No
+
+### Location Tag Category
+The default location tag category that ships with CFME does not allow multiple values so we need to delete and recreate it.  This is don in the Region configuration area of CFME.
+This tag will be populated with all the locations from Satellite and used to tag what locations can dbe used with each provider
+*   Delete location Tag Category
+*   Recreate location Tag Category
+    *   Name: location
+    *   Description: Location
+    *   Long Description: Location
+    *   Show in Console: Yes
+    *   Single Value: No
+    *   Capture C & U Data by Tag: No
 
 ### Operating System Tag Category
-
+This is a new tag that will need to be created and ipopulated with what operating systems are supported from within CFME
```

Comment:
This is a new tag category created and populated with operating systems values appropriate to your organization